draftlogs/7006_add.md

```diff
@@ -0,0 +1 @@
+ - Add property axis.ticklabelshiftx axis.ticklabelshifty for shifting the tick labels by a number of pixels [[#7006](https://github.com/plotly/plotly.js/pull/7006)]
```

Comment:
Please update the name of the attributes in the PR description as well as draft log.
Thank you!